tests/Unit/Console/ImportGreenlightTest.php

```diff
@@ -0,0 +1,318 @@
+        $rooms[]        = new GreenlightRoom(6, $users[5]->id, 'Test Room 6', 'abc-def-678');
+
+        $rooms[]        = new GreenlightRoom(7, $users[0]->id, 'Test Room 7', 'hij-klm-123', 123456, ['muteOnStart' => false,'requireModeratorApproval' => true,'anyoneCanStart' => false,'joinModerator' => true]);
+        $rooms[]        = new GreenlightRoom(8, $users[0]->id, 'Test Room 8', 'hij-klm-234', null, ['muteOnStart' => true,'requireModeratorApproval' => false,'anyoneCanStart' => true,'joinModerator' => false]);
+        $rooms[]        = new GreenlightRoom(9, 99, 'Test Room 9', 'hij-klm-456', 123456);
+        $rooms[]        = new GreenlightRoom(10, $users[0]->id, 'Test Room 10', $existingRoom->id);
+
+        // Create fake shared accesses
+        $sharedAccesses    = [];
+        $sharedAccesses[]  = new GreenlightSharedAccess(1, 1, 2);
+        $sharedAccesses[]  = new GreenlightSharedAccess(2, 1, 3);
+        $sharedAccesses[]  = new GreenlightSharedAccess(3, 1, 4);
+        $sharedAccesses[]  = new GreenlightSharedAccess(4, 1, 5);
+        $sharedAccesses[]  = new GreenlightSharedAccess(5, 1, 6);
+        $sharedAccesses[]  = new GreenlightSharedAccess(6, 4, 1);
+        $sharedAccesses[]  = new GreenlightSharedAccess(7, 9, 1);
+
+        // mock database connections with fake data
+        $this->fakeDatabase($roomAuth, new Collection($users), new Collection($rooms), new Collection($sharedAccesses));
+
+        // run artisan command and text questions and outputs
+        $this->artisan('import:greenlight localhost 5432 greenlight_production postgres 12345678')
+            ->expectsQuestion('What room type should the rooms be assigned to?', 'VL')
+            ->expectsQuestion('Prefix for room names:', $prefix)
+            ->expectsOutput('Importing users')
+            ->expectsOutput('3 created, 2 skipped (already existed)')
+            ->expectsOutput('LDAP import failed for the following 1 users:')
+            ->expectsOutput('+----------+----------+')
+            ->expectsOutput('| Name     | Username |')
+            ->expectsOutput('+----------+----------+')
+            ->expectsOutput('| John Doe | johnd    |') // user does not exist in fake ldap directory
+            ->expectsOutput('+----------+----------+')
+            ->expectsOutput('Importing rooms')
+            ->expectsOutput('7 created, 1 skipped (already existed)')
+            ->expectsOutput('Room import failed for the following 2 rooms, because no room owner was found:')
+            ->expectsOutput('+-------------+-------------+-------------+')
+            ->expectsOutput('| Name        | ID          | Access code |')
+            ->expectsOutput('+-------------+-------------+-------------+')
+            ->expectsOutput('| Test Room 4 | abc-def-456 |             |') // room owner the the user that as missing in the ldap
+            ->expectsOutput('| Test Room 9 | hij-klm-456 | 123456      |') // user was not found in greenlight DB
```

Comment:
A Test for already existing room would be nice!

Reply:
The existing room is the output "1 skipped (already existed)"